po/fr.po

```diff
@@ -3793,7 +3793,7 @@ msgstr "le fichier %s n'est pas inscriptible par l'utilisateur"
 
 #: trailer.c:965
 msgid "could not open temporary file"
-msgstr "impossible de créer un fichier temporaire"
+msgstr "impossible d'ouvrir un fichier temporaire"
```

Comment:
Je me suis posé la question aussi. Mais on ne sait pas de quel fichier il s'agit.
Dans la version anglaise, si on avait parlé d'un fichier spécifique (donc "le fichier temporaire"), n'aurions-vous pas eu "could not open the temporary file" ?